### PR TITLE
Add script to scale deployments

### PR DESCRIPTION
**Pull request by howardjohn**
This will autoscale service graphs between 1-3 replicas. Should give the same load as having a fixed 2 replicas

### CODE REVIEW

File: perf/load/templates/config-map.yaml [hunk 1]
```diff
@@ -46,7 +46,8 @@ data:
     export INGRESS_IP={{ .Values.ingress }}
 
     ${WD}/rollout.sh &
-    ${WD}/rotate.sh {{ .Values.serviceNamePrefix }}0 {{ .Values.domain }} 60m &
+    ${WD}/scaling.sh &
+    {{/*${WD}/rotate.sh {{ .Values.serviceNamePrefix }}0 {{ .Values.domain }} 60m &*/}}
```
Comment (mandarjog) [inline perf/load/templates/config-map.yaml]: what is `/*  .... */` doing here?

Comment (howardjohn) [reply perf/load/templates/config-map.yaml]: an accident, let me fix this

Comment (mandarjog) [reply perf/load/templates/config-map.yaml]: Ping.
I think this is great to get in quickly.

Comment (howardjohn) [reply perf/load/templates/config-map.yaml]: Fixed

**Pull request merged**